src/main/java/org/snomed/snowstorm/rest/ConceptController.java

```diff
@@ -535,6 +576,44 @@ public AsyncConceptChangeBatch getConceptBulkChange(@PathVariable String branch,
 		return findConceptsWithECL(">" + conceptId, form == Relationship.CharacteristicType.stated, branch, acceptLanguageHeader, 0, LARGE_PAGE.getPageSize()).getItems();
 	}
 
+	@GetMapping(value = "/browser/{branch}/concepts/ancestorPaths")
```

Comment:
Please add an `@ApiOperation` annotation with a `value` and some `notes` to explain what this does.

Reply:
Please change the mapping from `ancestorPaths` to `ancestor-paths`, we don't use camel case in the rest of the API.

Reply:
Changes made.